fusedrug/data/tokenizer/modulartokenizer/special_tokens.py

```diff
@@ -49,6 +49,8 @@
     "GENESEQ",  # Genetic sequence from AAs (SMILES)
     "INCREASE",  # Given a molecule and a property, generate a new molecule with increased property
     "DECREASE",  # Given a molecule and a property, generate a new molecule with decreased property
+    "DIFFUSION",  # Discrete diffusion task
```

Comment:
Maybe pick more meaningful names? like <DIFFUSION_TIMESTEP> instead of <TIMESTEP>?